google-cloud-firestore/src/main/java/com/google/cloud/firestore/StreamableQuery.java

```diff
@@ -265,47 +255,12 @@ public void onCompleted() {
       return result;
     } catch (Exception error) {
       span.end(error);
-      metricsContext.recordLatency(MetricType.END_TO_END_LATENCY, error);
       throw error;
     }
   }
 
-  class MonitoredStreamResponseObserver implements ApiStreamObserver<RunQueryResponse> {
```

Comment:
since E2Elatency is removed, it does not make sense to have a class just to record first response latency: https://github.com/googleapis/java-firestore/pull/1840/files#r1809028150